site/src/request_handlers/github.rs

```diff
@@ -5,15 +5,13 @@ use crate::github::{
 };
 use crate::load::SiteCtxt;
 
-use std::sync::Arc;
-
+use hashbrown::HashMap;
```

Comment:
why hashbrown and not std?

Reply:
No real reason, it was autoimported. But I don't see any disadvantage in using `hashbrown`, since we already depend on it, it doesn't really matter. We don't need DOS protection here xD